src/SFA.DAS.FindEmploymentSchemes.Contentful/Services/ContentService.cs

```diff
@@ -78,13 +89,11 @@ public async Task<IContent> UpdatePreview()
             if (_previewContentfulClient == null)
                 throw new ContentServiceException("Can't update preview content without a preview ContentfulClient.");
 
-            var previewContent = await Update(_previewContentfulClient);
```

Comment:
This tidy up has an unintended side effect.

`ContentService` is a singleton, so the property `PreviewContent` is also. That means that with this change, the preview content returned is not necessarily the preview content just fetched on the thread. I think it's safer to return the local variable instead.

Reply:
Addressed in latest commit